includes/definitions/discovery/zynos.yaml

```diff
@@ -10,3 +10,44 @@ modules:
               -
                   oid: sysMgmtCPU5SecUsage
                   num_oid: '.1.3.6.1.4.1.890.1.15.3.2.7.{{ $index }}'
+    sensors:
+        fanspeed:
+            options:
+                skip_values_lt: 0
+            data:
+                -
+                    oid: ZYXEL-HW-MONITOR-MIB::zyxelHwMonitorFanRpmTable
```

Comment:
You specify this additional mib at the top like:

`mib: ZYXEL-ES-COMMON:ZYXEL-HW-MONITOR-MIB`

Reply:
No problem, I've done that but left "ZYXEL-GS2200-24-MIB"
Hope that is alright.

Any advice on how to assign a custom ifAlias?
Ideally taking the $port_id [101 - 430] and insert a slash between the first integer and the second 2. ie 1/01 - 4/30

The ports appear as :- 
-   swp00 (actually 1/1, $port 101)
-   swp64 (actually 2/1, $port 201)
-   swp128 (actually 3/1, $port 301)
-   swp192 (actually 4/1, $port 401)

Currently the $port['ifAlias'] is empty

Cheers